### ISSUE TITLE
Could not deploy multiple services those set custom domain with `http.alias`

### ISSUE THREAD

**Issue by shgtkshruch** [aws/copilot-cli#2504]
Hi, there. I update copilot to 1.8.0 to set custom domain to multiple services.

```sh
$ copilot --version
copilot version: v1.8.0
```

I have 2 Load Balanced Web Services `frontend` and `rails` those set custom domain as `http.alias` field. 

```yml
name: frontend
type: Load Balanced Web Service

...

environments:
  dev:
    http:
      alias: dev-frontend.example.com
  staging:
    http:
      alias: staging-frontend.example.com
```

```yml
name: rails
type: Load Balanced Web Service

...

environments:
  dev:
    http:
      alias: dev-rails.example.com
  staging:
    http:
      alias: staging-rails.example.com
```

First, I deploy `frontend` service to `dev` environment (`copilot svc deploy --name frontend --env dev`), that enable access `dev-frontend.example.com`. Next, I deploy `rails` service to `dev` environment (`copilot svc deploy --name rails --env dev`), that occur error below.

```
- Creating the infrastructure for stack chronos-dev-rails                       [rollback complete]   [67.3s]
  The following resource(s) failed to create: [EnvControllerAction]. Rol                              
  lback requested by user.                                                                            
  - An EFS access point to handle POSIX permissions                             [not started]         
  - An Addons CloudFormation Stack for your additional AWS resources            [review in progress]  [630.3s]
    - A security group for your workload to access the DB cluster railscluster  [not started]         
  - Service discovery for your services to communicate within the VPC           [delete complete]     [0.0s]
  - Update your environment's shared resources                                  [update complete]     [72.1s]
    - Add an A-record to the hosted zone for the domain alias                   [update complete]     [0.0s]
    - Request and validate an ACM certificate for your domain                   [delete complete]     [67.8s]
  - An IAM Role for the Fargate agent to make AWS API calls on your behalf      [delete complete]     [1.4s]
  - A CloudWatch log group to hold your service logs                            [delete complete]     [0.0s]
  - An ECS service to run and maintain your tasks in the environment cluster    [not started]         
  - A target group to connect the load balancer to your service                 [delete complete]     [0.0s]
  - An ECS task definition to group your containers and run them on ECS         [not started]         
  - An IAM role to control permissions for the containers in your tasks         [not started]     
```

I confirm management console in CloudFormation, output this error below.

![スクリーンショット 2021-06-23 20 03 46](https://user-images.githubusercontent.com/5207601/123088421-abd8e780-d460-11eb-897f-ef79580a3359.png)

**Comment by huanjani** [comment]
Hi @shgtkshruch! It looks like you had a stack stuck in the cleanup state. Had you created it once before then tried to delete it? Can you try manually deleting the stack then trying again?

**Comment by iamhopaul123** [comment]
Hello @shgtkshruch before cleaning it up could you check the environment stack to see why the cleaning up failed? under the hood envcontroller updates the env stack. Thank you!

Also I tried to create two LB services and deploy to the same env using the alias `v1.example.com` and `v2.example.com` respectively. The second deployment was successful.

**Comment by iamhopaul123** [comment]
OK found the bug. Will prioritize and address this problem very soon. Thank you for reporting!

**Comment by iamhopaul123** [comment]
Heya we fixed this in our latest release [v1.8.2](https://github.com/aws/copilot-cli/releases/tag/v1.8.2). Thank you for reporting this issue ❤️
